config/config.go

```diff
@@ -29,7 +29,9 @@ rpc-scheme = "{{ .PlmntConfig.RPCScheme }}"
 rpc-wallet = "{{ .PlmntConfig.RPCWallet }}"
 reissuance-asset = "{{ .PlmntConfig.ReissuanceAsset }}"
 validator-address = "{{ .PlmntConfig.ValidatorAddress }}"
-distribution-address-inv = "{{ .PlmntConfig.DistributionAddrInv }}"
+distribution-address-early-inv = "{{ .PlmntConfig.DistributionAddrEarlyInv }}"
+distribution-address-investor = "{{ .PlmntConfig.DistributionAddrInvestor }}"
```

Comment:
Please leave this `distribution-address-inv = "{{ .PlmntConfig.DistributionAddrInv }}"` and move it one line up again to remove the diff. This is also how we use it in Ansible.

Reply:
The investor pool got split up. that's why we have a different name now. I also wanted to name it properly. so that there is less misunderstanding